tests/test_files_client.py

```diff
@@ -55,9 +55,10 @@ def test_list_folder(config):  # noqa: F811
 
     data = files_client.list_folder("test-upload/")
 
-    assert set(data.keys()) == set(src_to_dest.values()), (
-        "Failed to list files correctly"
-    )
+    remote_files = set(data.keys())
+    remote_files.discard("test-upload/.DS_Store")
+
```

Comment:
should also drop .DS_Store from src_to_dest because that will exist on macOS when run locally